network.js

```diff
@@ -18,7 +18,8 @@ function blockFrostAmountToValue(obj) {
         if (item.unit == "lovelace") {
             value = value.add(new Value(qty));
         } else {
-            let mph = MintingPolicyHash.fromHex(item.unit);
+            let policyID = `${item.unit}`.substring(0, 56);
+            let mph = MintingPolicyHash.fromHex(policyID);
 
             /** @type {number[]} */
             let token = [];
```

Comment:
you could write `let token = hexToBytes(item.unit.substring(56))` (I think that at the top of the `network.js`, `hexToBytes` is imported from the Helios library)

Reply:
Works. Thanks!